### ISSUE TITLE
Train ssd from scratch get bad result?

### ISSUE THREAD

**Issue by csldali** [pierluigiferrari/ssd_keras#160]
I follow the instructions to train ssd model from scratch in ssd300_training.ipynb , but finall i get bad result.
The Related parameters i set as follows:
batch_size=15(my computer not very good);
dataset path have corresponding modification （only use VOC）;
when i have trained 60 epoch, the loss is 5.06 and val loss is 4.8. after that i use ssd300_evaluation.ipynb to get MAP, it just only 0.34.
I try many time to find problem, but failed, can you get me some advice, thank you for you attention.

**Comment by pierluigiferrari** [comment]
The SSD300 training notebook should reproduce the original results as is. I can't really say much more than that, I've run the full training multiple times and gotten the reported results each time, so I wouldn't know what advice to give you.

The only thing you seem to have changed is the batch size, which should have at most a minor effect on the training progress.

The training and validation losses you report after 60 epochs are both where they should be, so that part looks good, but the mAP you get is inconsistent with those loss figures, so maybe double-check that the settings of the Evaluator are correct. And also just make a few predictions to get a rough qualitative evaluation of the model's performance after 60 epochs.

Another possibility that comes to mind is, maybe you're running Python 2 and that is causing issues, but that's a long shot.
